modules/sdk-core/src/bitgo/baseCoin/baseCoin.ts

```diff
@@ -284,6 +284,15 @@ export abstract class BaseCoin implements IBaseCoin {
     return false;
   }
 
+  /**
+   * Transform message to accommodate specific blockchain requirements.
+   * @param message the message to prepare
+   * @return string the prepared message.
+   */
+  prepareMessage(message: string): string {
```

Comment:
instead of adding a method to the parent class, I think you can have the eth class `implement` an interface which include the `prepareMessage` fn

adding a method which is used to be a few coins to the base class is not preferrable

note that this is a non-blocking comment

Reply:
what if this is needed for other coins in the future?

Reply:
Then maybe coin can implement the same interface. The base class is just returning the message - it's not performing any transformation or manipulation for the message. If the base class performed some sort of common manipulation that is needed for all coins then that is different.

Again, this is not a blocking comment. Just want to bring this up so that we avoid adding a new method that does nothing to the base class every time a child class needs a new method. Unfortunately with JS a class can only inherit from a single parent.

Reply:
added unit test and `spy` to check that message prefix was added.